### ISSUE TITLE
Dark Mode breaks the app

### ISSUE THREAD

**Issue by FraYoshi** [Artawower/orgnote#32]
### Thank you for the bug report

- [X] I am using the latest version of the `OrgNote` app.
- [X] I already checked other related issues
- [X] I have read the [WIKI](https://github.com/Artawower/orgnote/wiki)

### Bug description and steps to reproduce

Freah Install, Open it, set the interface to Dark Mode, and next time will not open again, or better, will open blank (grey)

Here a video showcasing the issue. Only Dark Mode, no any other single setting. (and auto does not work also, but that's for another issue)

### Debug info

OrgNote: 0.21.2
Language: en-US

Screen:
  Screen resolution: 448x998
  Screen color depth: 24
  Device pixel ratio: 2.25

Encryption:
  Type: disabled

Env:
  API URL: https://org-note.com/api/v1
  AUTH URL: https://org-note.com
  MODE: production

Quasar info:
  chrome: true
  version: 131.0.0.0
  versionNumber: 131
  android: true
  mobile: true
  webkit: true
  name: chrome
  platform: android
  capacitor: true
  nativeMobile: true
  nativeMobileWrapper: capacitor
 standalone: false

Device info:
  Model: Pixel 8 Pro
  Manufacturer: Google
  SDK version: 35
  Version: 15

### Additional context

Firat Time using it, first experience , I've just single switched to dark mode as a 1st thing.

**Comment by FraYoshi** [comment]
edit : added the video link

**Comment by FraYoshi** [comment]
Update: installed the github version, and dark-mode works without breaking the app.
Do not know if we should keep this open, as only affects the PlayStore version.

TLDR;
Playstore 0.21.2 broken;
Github 0.22.7 working.
